src/lengths.jl

```diff
@@ -0,0 +1,19 @@
+export gyroradius
+
+"""
+Calculate the radius of circular motion for a charged particle in a uniform magnetic field
+"""
+function gyroradius(B, mass_numb, Z, Vperp::Unitful.Velocity)
```

Comment:
This should be `ion_gyroradius`, and I think that the mass should be a `Unitful.Mass`. We can then think about providing an optional extension to compute the mass from a mass number. But I am a little cautious of this idea because it could cause confusion about what units are used.

Reply:
Agree! `gyroradius` is totally rewriiten in recent commits.